modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradeLayoutFeatures.kt

```diff
@@ -136,7 +152,25 @@ internal fun UpgradeLayoutFeatures(
             }
 
             if (canUpgrade) {
-                SubscribeButton(onClickSubscribe)
+                Column(
+                    modifier = Modifier,
+                    verticalArrangement = Arrangement.Center,
```

Comment:
```suggestion
```

I don't believe these two column attributes are needed?

Reply:
I put it to align the text below of the button